R/core.R

```diff
@@ -31,7 +31,7 @@ benchmark_run_iteration <- function(expr_before_benchmark,
         attach(loadNamespace("touchstone"), name = new_name)
         on.exit(detach(new_name, character.only = TRUE), add = TRUE)
         exprs_eval(expr_before_benchmark)
-        benchmark <- bench::mark(exprs_eval(...), memory = FALSE, iterations = 1)
+        benchmark <- bench::mark(exprs_eval(!!..1), memory = FALSE, iterations = 1)
```

Comment:
What is this doing? I know it's length one and we check it above but I prefer to solve this differently i possible. Maybe `!!!...` instead?

Reply:
this was necessary due to the double quoting, this is now different though